.github/workflows/sweeper.yaml

```diff
@@ -25,6 +25,12 @@ jobs:
           go-version-file: test/hack/resource/go.mod
           check-latest: true
           cache-dependency-path: "test/hack/resource/go.sum"
-      - run: go run main.go
+      # eu-north-1 contains the soak tests and resources should be sweept if they are older then 8 days
+      - run: |
+          if [[ "${{ matrix.region }}" == "eu-north-1" ]]; then
+            go run main.go --expiration 192h
+          else
+            go run main.go --expiration 12h
```

Comment:
Looks like the default is 12h. Can we just not set it here, or not make a default?

Reply:
I had asked to specify the expiration explicitly here. I think that it makes it clearer. Having a default probably still makes sense for anyone running the sweeper manually.